src/tools/crawler/network.rs

```diff
@@ -54,6 +65,13 @@ impl KnownNetwork {
         });
     }
 
+    /// Sets the node's state.
```

Comment:
`node state` can mean anything without further context
This could clear things up:
```suggestion
    /// Sets the node's connection state.
```

Reply:
Changed.